core/domain/param_domain.py

```diff
@@ -173,12 +203,19 @@ def from_dict(cls, param_change_dict):
             param_change_dict['customization_args']
         )
 
-    def get_value(self, context_params):
+    # The method `generate_value()` accepts first argument of type Any and it
+    # also returns values of type Any. So that's why argument context_params
+    # and the return type of `get_value()` is annotated as Any.
```

Comment:
I rephrased the comment a bit PTAL!!

Also, the type of `context_params` is annotated as `Dict[str, Any]` because there is no docstring for the argument, In tests we are providing only empty dictionary to the `context_params` and from the usage of function in codebase it is not clear that what type of values it contains.

https://github.com/oppia/oppia/blob/d9b12dced337ca5fb2ff57ce43fc11b21973b246/core/domain/param_domain_test.py#L115

https://github.com/oppia/oppia/blob/d9b12dced337ca5fb2ff57ce43fc11b21973b246/core/tests/test_utils_test.py#L373